samples/azure-quantum/hello-world/HW-ionq-cirq.ipynb

```diff
@@ -0,0 +1,346 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 👋🌍 Hello, world: Submit a Cirq job to IonQ\n",
+    "\n",
+    "In this notebook, we'll review the basics of Azure Quantum by submitting a simple *job*, or quantum program, to [IonQ](https://ionq.com/). We will use [Cirq](https://quantumai.google/cirq) to express the quantum job."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "source": [
+    "## Submit a simple job to IonQ using Azure Quantum\n",
+    "Azure Quantum provides several ways to express quantum programs. In this example we are using Cirq, but note that Q# and Qiskit are also supported. All code in this example will be written in Python.\n",
+    "\n",
+    "Let's begin. When you see a code block, hover over it and click the triangle play-button to execute it. To avoid any compilation issues, this should be done in order from top to bottom."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "source": [
+    "### 1. Connect to the Azure Quantum workspace\n",
+    "\n",
+    "To connect to the Azure Quantum service, construct an instance of the `AzureQuantumService`. Note that it's imported from `azure.quantum.cirq`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.quantum.cirq import AzureQuantumService\n",
+    "\n",
+    "service = AzureQuantumService(\n",
+    "    resource_id = \"\",\n",
```

Comment:
I'll have to add some tests to make sure that changing this text does not break our auth replace logic. Can that be added in future PR, since I need to test every sample with auth?

Reply:
Fair enough! A comment in the prose before would work as well, no need to modify the code cell itself, to be sure.

Reply:
To keep the hello worlds as simple as possible, I don't think we want to have local install instructions mixed in with the cloud sample, so we'd probably want to add a comment here that gets scrubbed by the auth replace logic.

We can take a look once this migration is done and make sure adding that doesn't break anything across all login types